internal/pkg/server/agent_test.go

```diff
@@ -449,6 +461,7 @@ func Test_Agent_configFromUnits(t *testing.T) {
 		assert.Equal(t, []string{"testhost:8080"}, cfg.Inputs[0].Server.Instrumentation.Hosts)
 		assert.Equal(t, "test", cfg.Inputs[0].Server.Instrumentation.GlobalLabels)
 		assert.Equal(t, "test-token", cfg.Output.Elasticsearch.ServiceToken)
+		assert.Equal(t, "0.01", cfg.Inputs[0].Server.Instrumentation.TransactionSampleRate)
```

Comment:
Should we keep a test case where the sampling rate is not specified?

Reply:
I can remove the sampling rate from one of the testcases or just add a new one with minimal apm config, I'm not sure I have a preference between the two. What would you rather have ?